library/core/src/sync/atomic.rs

```diff
@@ -1555,7 +1555,7 @@ impl<T> AtomicPtr<T> {
     /// previous pointer.
     ///
     /// This is equivalent to using [`wrapping_add`] and [`cast`] to atomically
-    /// perform `ptr = ptr.cast::<u8>().wrapping_add(val).cast::<T>()`.
+    /// perform `ptr = ptr.wrapping_byte_add(val)`.
```

Comment:
Given the text in the previous line here, would it be better to leave this alone?  Or should the references in the previous line point to something else?

Oh, the references are to pointer methods (down on line 1568).  So maybe update the cross-reference to `wrapping_byte_add` too?

Reply:
I think `wrapping_byte_add` conveys the semantics of this operation better (and it's of course literally the same as the old code), so I think the references should be updated.